### ISSUE TITLE
Windows magnifier support

### ISSUE THREAD

**Issue by ashkan-saeedi-mazdeh** [slynch8/10x#2932]
Hi
I use windows magnifier and it tracks the cursor. I thought since this is still a software with not a huge user base, it probably does not have accessibility support but to my surprise , all lines were being read correctly with my text to speech.
I don't know what UI framework you are using to make this and I cannot guarantee I'll purchase, however visual studio + VAX for unreal is a bit slow even on my beast of a  machine so if this is really instant which I'll try and if it can automatically include files for an unreal class/method then I'll possibly switch to it.

I thought depending on the UI framework, since you have accessibility already, supporting magnifier cursor tracking might be easy. Perhaps an api call to report the coordinates which the magnifier should go to.

    Also thanks for making this accessible. This is not a financial decision, good of you to do it.

**Comment by ashkan-saeedi-mazdeh** [comment]
Thank you. Appreciate you caring about this.

**Comment by slynch8** [comment]
I just tried this and I'm not sure what it's supposed to do? It's magnifying whatever is on screen as I mouse over, and reading the text under the cursor (with Help Menu -> Text to speech enabled)

**Comment by ashkan-saeedi-mazdeh** [comment]
Try increasing the zoom so there is not much visible in the magnified area. then type some text in visual studio or notepad and see the zoomed area moving with the cursor.

Now type some text in 10x editor and see the magnified area does not move with the cursor in the text area/text box.
